pkg/e2e/osd/dedicatedadmin.go

```diff
@@ -269,6 +271,37 @@ var _ = ginkgo.Describe(dedicatedAdminTestName, func() {
 			err := manageSubscriptions(ctx, namespaceList, h)
 			Expect(err).NotTo(HaveOccurred())
 		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// dedicated-admin user can patch consoles.operator.openshift.io CR
+		util.GinkgoIt("ded-admin user can patch consoles.operator.openshift.io CR", func(ctx context.Context) {
+			// Impersonate ded-admin
+			h.Impersonate(rest.ImpersonationConfig{
+				UserName: "dummy-admin@redhat.com",
+				Groups: []string{
+					"dedicated-admin",
+				},
+			})
+			defer func() {
+				h.Impersonate(rest.ImpersonationConfig{})
+			}()
+
+			patchData := []byte(fmt.Sprintf(`{"spec":{"plugins":["test"]}}`))
+
+			_, err := h.Dynamic().Resource(schema.GroupVersionResource{
+				Group: "operator.openshift.io", Version: "v1",
+				Resource: "consoles",
+			}).Patch(ctx, "cluster", types.MergePatchType, patchData, metav1.PatchOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			// revret the changes
+			patchEmpty := []byte(fmt.Sprintf(`{"spec":{"plugins":[""]}}`))
+
+			_, err1 := h.Dynamic().Resource(schema.GroupVersionResource{
```

Comment:
should this be named as `err` as well?

Reply:
This has been addressed in my latest commit.